coil-svg/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+import coil.Library
+import coil.compileSdk
+import coil.minSdk
+import coil.targetSdk
+import org.jetbrains.dokka.gradle.DokkaAndroidTask
+
+plugins {
+    id("com.android.library")
+    id("com.vanniktech.maven.publish")
+    id("kotlin-android")
+    id("org.jetbrains.dokka-android")
+    id("org.jlleitschuh.gradle.ktlint")
+}
+
+android {
+    compileSdkVersion(project.compileSdk)
+    defaultConfig {
+        minSdkVersion(project.minSdk)
+        targetSdkVersion(project.targetSdk)
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+    libraryVariants.all {
+        generateBuildConfigProvider?.configure { enabled = false }
+    }
+    testOptions {
+        unitTests.isIncludeAndroidResources = true
+    }
+}
+
+afterEvaluate {
+    tasks.withType<DokkaAndroidTask> {
```

Comment:
This was copied over from `coil-gif`, but I don't think it's required.  Can you confirm?

Reply:
It's required to create the API docs for https://coil-kt.github.io/coil/.